pkg/metrics/metrics.go

```diff
@@ -56,8 +58,34 @@ func RegisterMetrics() error {
 	return nil
 }
 
-// UpdatePlaceholderGauge ...
-func UpdatePlaceholderGauge() {
+// UpdateMetrics updates all the metrics ever 5 minutes
+func UpdateMetrics() {
 
-	metricPlaceholder.With(prometheus.Labels{"name": "deadmanssnitch-operator"}).Set(float64(1))
+	d := time.Tick(5 * time.Minute)
```

Comment:
Want to be sure we're not going to hit rate limits, but can't find enough info on what those limits are.  There's mention of a "Rate Limited" response here https://deadmanssnitch.com/docs/api/v1 but I can't find anything more specific.  We have to keep in mind we have multiple clusters running this operator and can expect that to scale over time.

I recommend we find out more before we merge this in.

/hold